vmdb/app/models/ems_refresh/parsers/openstack_infra.rb

```diff
@@ -133,7 +133,7 @@ def parse_host(host, indexed_servers, _indexed_hosts_ports, indexed_resources, c
           :vmm_vendor           => 'RedHat',
           :vmm_product          => identify_product(indexed_resources, host.instance_uuid),
           :ipaddress            => ip_address,
-          :hostname             => ip_address,
+          :hostname             => cloud_host_attributes.try(:[], :host_name) || ip_address,
```

Comment:
@Ladas is this a hash?  Can we do `cloud_host_attributes.fetch(:host_name, ip_address)` to use the host_name if found, otherwise, default to ipaddress?